Apps/Medication/src/Delegates/IHNClientDelegate.cs

```diff
@@ -28,10 +28,11 @@ public interface IHNClientDelegate
         /// Retrieves a list of MedicationStatements that for the given phn.
         /// </summary>
         /// <param name="phn">The patient phn.</param>
+        /// <param name="protectiveWord">The clients protective word.</param>
         /// <param name="userId">The user id of the request.</param>
         /// <param name="ipAddress">The ip address of the request.</param>
         /// <returns>A List of MedicationStatement models.</returns>
-        Task<HNMessage<List<MedicationStatement>>> GetMedicationStatementsAsync(string phn, string userId, string ipAddress);
+        Task<HNMessage<List<MedicationStatement>>> GetMedicationStatementsAsync(string phn, string protectiveWord, string userId, string ipAddress);
```

Comment:
kinda nit: how about using a different method for calling with a protective word
```
Task<HNMessage<List<MedicationStatement>>> GetMedicationStatementsAsync(string phn, string userId, string ipAddress);
Task<HNMessage<List<MedicationStatement>>> GetProtectedMedicationStatementsAsync(string phn, string protectiveWord, string userId, string ipAddress);
```

That way the changes would only cascade to the methods that need it, and if not call the unprotected one.